config/database.yml

```diff
@@ -21,9 +21,12 @@ default: &default
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: 5
   host: localhost
-  username: root
   database: commons
 
+production:
+  <<: *default
+  username: root
```

Comment:
Follow up Q: why is it necessary to include at all if there is an `ENV['DATABASE_URL']` present? On the chance that it isn't present on prod?